webapp/src/app/shared/phaser/map-pan.ts

```diff
@@ -61,7 +61,7 @@ export class MapPan extends Phaser.GameObjects.GameObject {
 		
 		let zoom = delta > 0 ? 0.8 : 1.25;
 		zoom *= cam.zoom;
-		if (zoom > 0.4 && zoom < 8) {
+		if (zoom > 0.4 && zoom < 50) {
```

Comment:
This change is not mentioned in the change log and I don't see any particular reason for it. Was the change intentional?

Reply:
forgot to add it in the changelog. It's for high res displays. The editor tries to display everything pixel perfect, so with a max zoom of 8 it is roughly comparable to running crosscode on a 4k monitor at full screen, not actually zooming into the map.